### ISSUE TITLE
Not working on hassio.

### ISSUE THREAD

**Issue by Stump2939** [james-fry/hassio-addons#15]
I cannot seem to get any data coming through on my hassio install. Here is my config/log when i start the addon:
 
Starting RTL_433 with parameters:
MQTT Host = hassio.local
MQTT User =
MQTT Password =
MQTT Topic = Hassio/433/Honeywell
RTL_433 Protocol = 70
RTL_433 Frequency = 345000000
RTL_433 Gain = 0
RTL_433 Frequency Offset = 0
Registering protocol [1] "Honeywell Door/Window Sensor"
Registered 1 out of 102 device decoding protocols
Found 1 device(s)
trying device  0:  Realtek, RTL2838UHIDIR, SN: 00000001
Detached kernel driver
Found Rafael Micro R820T tuner
Using device 0: Generic RTL2832U OEM
Exact sample rate is: 250000.000414 Hz
[R82XX] PLL not locked!
Sample rate set to 250000.
Bit detection level set to 0 (Auto).
Tuner gain set to Auto.
Reading samples in async mode...
Tuned to 345000000 Hz.
Allocating 15 zero-copy buffers
Failed to allocate zero-copy buffer for transfer 0
Falling back to buffers in userspace

I was able to use a separat pi running rasbian and I managed to get use the rtl software to receive data from my sensors so I know the hardware does function. I just dont know enough about it to figure out why the dongle will not work with Hassio. Also is there any way to see if the addon is receiving a wireless signal before it sends out mqtt data?

**Comment by james-fry** [comment]
What is the log when it was working on the separate pi?

**Comment by Stump2939** [comment]
Sorry I didn't specify but I didn't have it installed with hassio I used
some form of rtl_433 and installed it on raspian via command line. I'm not
sure what exactly it was I I stalled it was a while ago. But I do remember
selecting the Honeywell protocol and it was picking up my motion sensors
and displaying the data. But I could not get the mqtt portion to work so I
have up. But since my SDR dongle worked and I have hassio running on
another box I was hoping to put the dongle in my hassio box and go from
there. I just don't know how to find out if I'm receiving anything.

**Comment by Stump2939** [comment]
Update... I found in the rtl2mqtt.sh file a line commented out:
set -x  ## uncomment for MQTT logging...
Once i turn that line on I was able to refresh my log screen and it turns out everything was working. The problem was the mqtt subject. I was watching "Hassio/433/Honeywell" which was not where the data was being published. The logs showed the proper topics... Eg "MQTT_PATH=Hassio/433/Honeywell/Honeywell_Door/Window_Sensor/985335".... So I hope this helps someone someone else.
